apps/etterlatte-behandling/src/main/kotlin/sak/SakRoutes.kt

```diff
@@ -8,10 +8,19 @@ import io.ktor.server.routing.delete
 import io.ktor.server.routing.get
 import io.ktor.server.routing.route
 import no.nav.etterlatte.Kontekst
+import no.nav.etterlatte.behandling.GenerellBehandlingService
+import no.nav.etterlatte.behandling.toBehandlingSammendrag
+import no.nav.etterlatte.grunnlagsendring.GrunnlagsendringsListe
+import no.nav.etterlatte.grunnlagsendring.GrunnlagsendringshendelseService
 import no.nav.etterlatte.inTransaction
+import no.nav.etterlatte.libs.common.behandling.BehandlingListe
 import no.nav.etterlatte.libs.common.behandling.SakType
 
-internal fun Route.sakRoutes(sakService: SakService) {
+internal fun Route.sakRoutes(
+    sakService: SakService,
+    generellBehandlingService: GenerellBehandlingService,
```

Comment:
Kanskje vi kan flytte `GenerellBehandlingService` og `GrunnlagsendringshendelseService` inn i `SakService`?

Reply:
Kan gjøre det. Litt uklar over hva som er mest vanlig her - noen service har andre service mens andre steder er det route som har de. Ser ut som det er mest en smakssak.

Kan flytte de to inn.

Reply:
Dette var ikke direkte enkel. GenerellBehandlingService ender opp med en SakService inn i seg - så å flytte den inn gir en sirkular avhengighet

Reply:
Tenker vi bare kan la det være som det er nå, siden det ikke er konsekvent fra før

Reply:
Da kan du bare ha det sånn ja!